Azure.ResourceManager.Core/SubscriptionContainer.cs

```diff
@@ -7,7 +7,7 @@
 namespace Azure.ResourceManager.Core
 {
     /// <summary>
-    /// Subscription Container Operationss
+    /// GetSubscriptionOperations Container Operationss
```

Comment:
I think this comment updated wasn't intended?

Reply:
No that was the automate rename refactoring that updated this comment incorrectly.